.github/workflows/bump-plugins-submodule.yaml

```diff
@@ -34,6 +34,11 @@ jobs:
           app-id: ${{ vars.TENZIR_AUTOBUMPER_APP_ID }}
           private-key: ${{ secrets.TENZIR_AUTOBUMPER_APP_PRIVATE_KEY }}
           owner: ${{ github.repository_owner }}
+      - uses: crazy-max/ghaction-import-gpg@v6
+        with:
+          gpg_private_key: ${{ secrets.TENZIR_BOT_GPG_SIGNING_KEY }}
```

Comment:
How does this work if it's the app that creates the commit and this is the gpg private key of the `tenzir-bot` account?

Reply:
Right, we also need to set the user/email so that it fits with the gpg key.

(the app should be orthogonal since its only used for github permissions, but the signing is for the commit object in git itself)